src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/WriteHelper.cs

```diff
@@ -207,7 +207,10 @@ public async Task<int> CopyLedgerTransaction(ICollection<LedgerTransaction> enti
             await writer.WriteAsync(lt.EngineReceipt.ErrorMessage, NpgsqlDbType.Text, token);
             await writer.WriteAsync(lt.EngineReceipt.Output, NpgsqlDbType.Jsonb, token);
             await writer.WriteAsync(lt.EngineReceipt.NextEpoch, NpgsqlDbType.Jsonb, token);
-            await writer.WriteAsync(lt.EngineReceipt.Events, NpgsqlDbType.Jsonb, token);
+            await writer.WriteAsync(lt.EngineReceipt.EventsSbor, NpgsqlDbType.Array | NpgsqlDbType.Bytea, token);
+            await writer.WriteAsync(lt.EngineReceipt.EventsSchemaHash, NpgsqlDbType.Array | NpgsqlDbType.Bytea, token);
+            await writer.WriteAsync(lt.EngineReceipt.EventsTypeIndex, NpgsqlDbType.Array | NpgsqlDbType.Integer, token);
+            await writer.WriteAsync(lt.EngineReceipt.EventsTypeKind, "key_type_kind[]", token);
```

Comment:
Have you tried this with EF? Looks like something that might not be supported by EF.